boards/xtensa/esp32/esp32-devkitc/configs/coremark/defconfig

```diff
@@ -21,39 +20,22 @@ CONFIG_ARCH_STACKDUMP=y
 CONFIG_ARCH_XTENSA=y
 CONFIG_BENCHMARK_COREMARK=y
 CONFIG_BOARD_LOOPSPERMSEC=16717
-CONFIG_BUILTIN=y
-CONFIG_DEBUG_FULLOPT=y
-CONFIG_DEBUG_SYMBOLS=y
+CONFIG_DEBUG_CUSTOMOPT=y
+CONFIG_DEBUG_OPTLEVEL="-O3"
 CONFIG_ESP32_UART0=y
-CONFIG_EXAMPLES_HELLO=y
 CONFIG_FS_PROCFS=y
-CONFIG_HAVE_CXX=y
-CONFIG_HAVE_CXXINITIALIZE=y
 CONFIG_IDLETHREAD_STACKSIZE=3072
-CONFIG_INIT_ENTRYPOINT="nsh_main"
+CONFIG_INIT_ENTRYPOINT="coremark_main"
 CONFIG_INIT_STACKSIZE=3072
 CONFIG_INTELHEX_BINARY=y
-CONFIG_MM_REGIONS=4
```

Comment:
`MM_REGIONS` should be set to 3 on ESP32 in order to fully utilize the Internal RAM as heap.

Reply:
My bad, I forgot to change it.